### DISCUSSION TITLE
How do I align buttons in the NavigationAppBar to the right

### DISCUSSION THREAD

**Discussion by FengZi-lv** [bdlukaa/fluent_ui#804] [Q&A]
How do I align buttons in the NavigationAppBar to the right

![image](https://user-images.githubusercontent.com/59948673/230707815-dd7da877-2287-4166-8e40-d5e1b128ac0d.png)

```dart
NavigationView(
      appBar: NavigationAppBar(
        title: Text(
          'title',
          style: typography.body,
        ),
        automaticallyImplyLeading: false,
      ),
      pane: NavigationPane(
                 // ...
           )
      // ...
)
```

Thanks your answer!

**Comment by bdlukaa** [comment]
The `NavigationView` button is not available to be changed by a property. What you can do is set `automaticallyImplyLeading` to `false` in the app bar and manually create the button in the `title` property. You can open the navigation view minimal pane mode using a `GlobalKey<NavigatorState>`
